### ISSUE TITLE
Gibbon::MailChimpError: the server responded with status 405

### ISSUE THREAD

**Issue by trystant** [BOWiki/BOW#163]
View details in Rollbar: [https://rollbar.com/trystant/BOW/items/197/](https://rollbar.com/trystant/BOW/items/197/)

```

Faraday::ClientError: the server responded with status 405
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/faraday-0.9.1/lib/faraday/response/raise_error.rb", line 13, in on_complete
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/faraday-0.9.1/lib/faraday/response.rb", line 9, in block in call
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/faraday-0.9.1/lib/faraday/response.rb", line 57, in on_complete
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/faraday-0.9.1/lib/faraday/response.rb", line 8, in call
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/faraday-0.9.1/lib/faraday/rack_builder.rb", line 139, in build_response
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/faraday-0.9.1/lib/faraday/connection.rb", line 377, in run_request
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/faraday-0.9.1/lib/faraday/connection.rb", line 177, in patch
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/gibbon-2.0.0/lib/gibbon/api_request.rb", line 24, in patch
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/gibbon-2.0.0/lib/gibbon/request.rb", line 34, in update
  File "/Users/dariusgoore/development/blackops/app/models/user.rb", line 59, in add_to_mailchimp
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activesupport-4.2.3/lib/active_support/callbacks.rb", line 430, in block in make_lambda
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activesupport-4.2.3/lib/active_support/callbacks.rb", line 237, in call
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activesupport-4.2.3/lib/active_support/callbacks.rb", line 237, in block in halting
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activesupport-4.2.3/lib/active_support/callbacks.rb", line 504, in call
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activesupport-4.2.3/lib/active_support/callbacks.rb", line 504, in block in call
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activesupport-4.2.3/lib/active_support/callbacks.rb", line 504, in each
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activesupport-4.2.3/lib/active_support/callbacks.rb", line 504, in call
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activesupport-4.2.3/lib/active_support/callbacks.rb", line 88, in run_callbacks
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activemodel-4.2.3/lib/active_model/validations/callbacks.rb", line 113, in run_validations!
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activemodel-4.2.3/lib/active_model/validations.rb", line 334, in valid?
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/validations.rb", line 58, in valid?
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/validations.rb", line 83, in perform_validations
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/validations.rb", line 37, in save
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/attribute_methods/dirty.rb", line 21, in save
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/transactions.rb", line 286, in block (2 levels) in save
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/transactions.rb", line 351, in block in with_transaction_returning_status
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/connection_adapters/abstract/database_statements.rb", line 213, in block in transaction
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/connection_adapters/abstract/transaction.rb", line 184, in within_new_transaction
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/connection_adapters/abstract/database_statements.rb", line 213, in transaction
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/transactions.rb", line 220, in transaction
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/transactions.rb", line 348, in with_transaction_returning_status
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/transactions.rb", line 286, in block in save
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/transactions.rb", line 301, in rollback_active_record_state!
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activerecord-4.2.3/lib/active_record/transactions.rb", line 285, in save
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/devise-3.4.1/app/controllers/devise/registrations_controller.rb", line 19, in create
  File "/Users/dariusgoore/development/blackops/app/controllers/users/registrations_controller.rb", line 13, in create
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/actionpack-4.2.3/lib/action_controller/metal/implicit_render.rb", line 4, in send_action
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/actionpack-4.2.3/lib/abstract_controller/base.rb", line 198, in process_action
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/actionpack-4.2.3/lib/action_controller/metal/rendering.rb", line 10, in process_action
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/actionpack-4.2.3/lib/abstract_controller/callbacks.rb", line 20, in block in process_action
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activesupport-4.2.3/lib/active_support/callbacks.rb", line 115, in call
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activesupport-4.2.3/lib/active_support/callbacks.rb", line 115, in call
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activesupport-4.2.3/lib/active_support/callbacks.rb", line 553, in block (2 levels) in compile
  File "/Users/dariusgoore/development/blackops/.bundle/gems/ruby/2.2.0/gems/activesupport-4.2.3/lib/

**Comment by campgurus** [comment]
I have sent a message to MailChimp support on this ticket, so hopefully they will have a fix.

**Comment by trystant** [comment]
What was the solution to this problem?

On Mon, Feb 29, 2016 at 6:09 AM, campgurus <notifications@github.com> wrote:

> Closed #163 <https://github.com/BOWiki/BOW/issues/163>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BOWiki/BOW/issues/163#event-570893394>.
>



-- 

Mark Nyonmark@grandkru.com

blog.grandkru.com

github.com/trystant
